protobuf/src/proto/shared/NullableString.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+message NullableString {
+	oneof string {
```

Comment:
Not sure if you want feedback on the proto files but the `oneof` block needs an extra space for it to be 2 spaces indented.

Reply:
These files were pull from https://github.com/azure/azure-functions-language-worker-protobuf as is, so I will not update them there. These change will have to be made at that repo.